acmg_db/utils/cnv_utils.py

```diff
@@ -269,16 +269,20 @@ def calculate_acmg_class(score):
 	print(score)
```

Comment:
get rid of print statement

Reply:
Fixed in https://github.com/AWGL/variant_classification_DB/pull/60/commits/aa56f2968168203e67261b3362e89b6e52d1497e